ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/Clang_64bit/tasks/main.yml

```diff
@@ -21,7 +21,7 @@
   tags: clang_64bit
 
 - name: Install Clang 64bit
-  win_command: 'c:\temp\LLVM-7.0.0-win64.exe /S'
+  win_command: 'c:\temp\llvm-7.0.0-win64.exe /S'
```

Comment:
This needs to be an unzip operation on the zip file now that the download has been changed